sourcegraph/sourcegraph
# notebook: command palette

Fixes #32839

https://user-images.githubusercontent.com/6417322/159427335-bd611a3d-9e43-4a02-94fe-3b95b6637c36.mov

## Test plan



* Open a notebook
* Start typing in the command palette input
* You should get a couple of options to automatically create blocks
* If you type / at the start, you should see all possible options
  * If you type /<text> those options should be filtered
* If you paste in a file URL you should get a single option to add a file block parsed from the URL
* Integration tests cover most of the use cases
